sdk/cosmos/azure-cosmos-benchmark/pom.xml

```diff
@@ -14,7 +14,7 @@ Licensed under the MIT License.
 
   <groupId>com.azure</groupId>
   <artifactId>azure-cosmos-benchmark</artifactId>
-  <version>4.0.1-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos-benchmark;current} -->
+  <version>4.0.1-beta.3</version> <!-- {x-version-update;com.azure:azure-cosmos-benchmark;current} -->
```

Comment:
This is an automated process for release version bump.
also we haven't released 4.0.1-beta.2 I think.
https://mvnrepository.com/artifact/com.azure/azure-cosmos